plugins/ban/events/baldban.py

```diff
@@ -23,6 +23,5 @@ async def execute(
             await ctx.send(
                 "https://media.discordapp.net/attachments/791335982666481675/979052868915064862/Chauve_qui_peut_.png"
             )
-        return True
-    else:
-        return False
+    
+    return False # allow Leirof to also get reverse ban and other fun things
```

Comment:
Euh... it was already the case

Reply:
(pourquoi je parle anglais ?)

Reply:
C'est le cas sur master et beta, là ascpial à rewrite la quasi-totalité du plugin et son rewrite faisait que le baldban était systématique, il le corrige ici du coup

Reply:
Okok, gotcha